api/tests/integration/infrastructure/repositories/campaign-collective-result-repository-test.js

```diff
@@ -310,81 +310,92 @@ describe('Integration | Repository | Service | Campaign collective result reposi
           const userWithCampaignParticipationCharlie = _createUserWithCampaignParticipation('Charlie', campaignId, campaignParticipationShareDate);
           const charlieId = userWithCampaignParticipationCharlie.userId;
 
-          // Dan (did not shared its campaign participation)
+          // Dan (did not share his campaign participation)
```

Comment:
`Did not share his campaign participation`

Reply:
Oui ?

Reply:
Une faute d'orthographe peux-être ?

Reply:
Ben je la vois pas justement ? J'ai du caca dans l'œil ou bien ?

Reply:
Y'a eu une modification entre temps :)

Reply:
good.